python/hsfs/core/feature_monitoring_config_api.py

```diff
@@ -16,7 +16,7 @@
 
 from typing import List, Optional
 from hsfs import client
-import hsfs.core.feature_monitoring_config as fmc
+from hsfs import feature_monitoring_config as fmc
```

Comment:
Let's wait until we are more or less ready to release to move the FM config out of core. I am not sure FM result is a class that will be very relevant to the user so we might not want to move it out of core.

Reply:
Alright :) sorry, I thought the fact of being in core was not intentional